src/components/layouts/pages/jobRouter/JobRouter.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import AppPages from '../../../AppPages';
+import JobStatus from '../status/JobStatus';
+import Results from '../results/Results';
+import Inspect from '../inspect/Inspect';
+
+function JobRouter() {
+    return (
+        <Switch>
```

Comment:
The goal of the common Job router was to accumulate common logic like redirecting to NotFount page if job doesn't exist and so on. So let's do that improvement already - revise what should be the common logic and what should be specific for certain pages.
Note that summary and inspect have some common logic as well, so please think how we can improve and keep in one place as well in scope of this PR.

Reply:
As discussed let's move refactoring out of this PR